### ISSUE TITLE
Better documentation: describe usage in template

### ISSUE THREAD

**Issue by PetrDlouhy** [AndrewIngram/django-extra-views#169]
I spent few hour figuring out, why inline formsets doesn't work. The cause was that I used `{{ form }}` instead of `{{ formset }}` in template. I didn't find any mention about this in the docs. This is worth of describing somewhere.

**Comment by PetrDlouhy** [comment]
The usage for `WithInlinesView`s is even more tricky. I wouldn't have a chance to guess it without the example in tests:
```
{{ form }}

{% for formset in inlines %}

{{ formset }}

{% endfor %}
```

**Comment by sdolemelipone** [comment]
Addressed in #178.
